datasets/wrbsc/README.md

```diff
@@ -0,0 +1,147 @@
+---
+annotations_creators: 
+- expert-generated
+language_creators: 
+- found
+languages: 
+- pl
+licenses: 
+- cc-by-sa-3.0
+multilinguality: 
+- monolingual
+size_categories: 
+- 1K<n<10K
+source_datasets: 
+- original
+task_categories: 
+- text-classification
+task_ids: 
+- semantic-similarity-classification
+---
+
+# Dataset Card for wrbsc
+
+## Table of Contents
+- [Dataset Description](#dataset-description)
+  - [Dataset Summary](#dataset-summary)
+  - [Supported Tasks](#supported-tasks-and-leaderboards)
+  - [Languages](#languages)
+- [Dataset Structure](#dataset-structure)
+  - [Data Instances](#data-instances)
+  - [Data Fields](#data-instances)
+  - [Data Splits](#data-instances)
+- [Dataset Creation](#dataset-creation)
+  - [Curation Rationale](#curation-rationale)
+  - [Source Data](#source-data)
+  - [Annotations](#annotations)
+  - [Personal and Sensitive Information](#personal-and-sensitive-information)
+- [Considerations for Using the Data](#considerations-for-using-the-data)
+  - [Social Impact of Dataset](#social-impact-of-dataset)
+  - [Discussion of Biases](#discussion-of-biases)
+  - [Other Known Limitations](#other-known-limitations)
+- [Additional Information](#additional-information)
+  - [Dataset Curators](#dataset-curators)
+  - [Licensing Information](#licensing-information)
+  - [Citation Information](#citation-information)
+
+## Dataset Description
+
+- **Homepage:** https://clarin-pl.eu/dspace/handle/11321/305
+- **Repository:** https://clarin-pl.eu/dspace/handle/11321/305
+- **Paper:** [Needs More Information]
+- **Leaderboard:** [Needs More Information]
+- **Point of Contact:** [Needs More Information]
+
+### Dataset Summary
+
+WUT Relations Between Sentences Corpus contains 2827 pairs of related sentences. Relationships are derived from Cross-document Structure Theory (CST), which enables multi-document summarization through identification of cross-document rhetorical relationships within a cluster of related documents. Every relation was marked by at least 3 annotators.
+
+### Supported Tasks and Leaderboards
+
+[Needs More Information]
+
+### Languages
+
+Polish
+
+## Dataset Structure
+
+### Data Instances
+
+[Needs More Information]
+
+### Data Fields
+
+- `sentence1`: a `string` feature.
+- `sentence2`: a `string` feature.
+- `relationship`: a `string` feature.
```

Comment:
can you list the values this field can take and describe their meanings please ?